WindDirectionControl/WindDirectionControl.js

```diff
@@ -22,11 +22,15 @@ import Skinnable from '../Skinnable';
 
 import css from './WindDirectionControl.module.less';
 
+// Children supplied to ArcPicker/WindDirectionControl. The number of items in this array represents the number of
+// Arcs rendered on the screen.
+const children = ['airDown', 'airRight', 'airUp'];
+
 /**
  * An Agate component for displaying Wind Direction Control {@link agate/WindDirectionControl}.
  *
- * @class WindDirectionControl
- * @memberof agate/WindDirectionControl
+ * @class WindDirectionControlBase
+ * @extends agate/ArcPicker.ArcPicker
```

Comment:
I might guide you incorrectly.
My intention is using memberof  and extends  together.
```
 * @memberof agate/WindDirectionControl
 * @extends agate/ArcPicker.ArcPicker
```

Reply:
Oh, sorry. It should be okay now 😅